test/system/700-play.bats

```diff
@@ -1035,3 +1035,72 @@ spec:
     run_podman rm -f -t 0 -a
     run_podman rmi automount_test
 }
+
+@test "podman ply with multi automount volume" {
+    cat >$PODMAN_TMPDIR/Containerfile1 <<EOF
+FROM $IMAGE
+RUN mkdir /test1 /test_same
+RUN touch /test1/a /test1/b /test1/c
+RUN echo "I am from test1 image" > /test_same/hello_world
+VOLUME /test1
+VOLUME /test_same
+EOF
+
+    cat >$PODMAN_TMPDIR/Containerfile2 <<EOF
+FROM $IMAGE
+RUN mkdir /test2 /test_same
+RUN touch /test2/asdf /test2/ejgre /test2/lteghe
+RUN echo "I am from test2 image" > /test_same/hello_world
+VOLUME /test2
+VOLUME /test_same
+EOF
+
+    run_podman build -t test1 -f $PODMAN_TMPDIR/Containerfile1
+    run_podman build -t test2 -f $PODMAN_TMPDIR/Containerfile2
+
+    fname="/$PODMAN_TMPDIR/play_kube_wait_$(random_string 6).yaml"
+    cat >$fname <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
```

Comment:
Do not hardcore the names please, see 
https://github.com/containers/podman/pull/23280 and https://github.com/containers/podman/pull/23275

I know this is the case today but we must stop doing that.

Reply:
`700-play` is next on my list for refactoring. @ypu if you wait maybe a week or so, until my (not-yet-filed) PR merges, you will have a much better base to work with.

Reply:
Thanks @Luap99 and @edsantiago . Will wait for Ed’s PR for refactoring the script.